package.json

```diff
@@ -30,12 +30,16 @@
     "@angular/platform-browser": "~9.1.7",
     "@angular/platform-browser-dynamic": "~9.1.7",
     "@angular/router": "~9.1.7",
+    "aframe": "^1.0.4",
+    "awesome-typescript-loader": "^5.2.1",
     "d3": "^5.16.0",
     "luxon": "^1.24.1",
     "random": "^2.2.0",
     "rxjs": "~6.5.4",
+    "source-map-loader": "^1.0.0",
```

Comment:
I think both `awesome-typescript-loader` and `source-map-loader` can be removed.
(You can run `npm uni awesome-typescript-loader source-map-loader` to remove the dependency tree from `package-lock.json`.)